advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx

```diff
@@ -1,12 +1,13 @@
 ---
-title: 'Partner Information'
+title: '1. Partner Information'
 description: 'Providing a general overview of Thales and the CipherTrust Transparent Encryption product'
 
 ---
+
 | &nbsp;     | &nbsp; |
 | ----------- | ----------- |
 | **Partner Name**      | Thales       |
 | **Partner Product**   |  CipherTrust Transparent Encryption       |
 | **Web Site**      |    https://cpl.thalesgroup.com/encryption    |
-| **Version & Platform**   |  7.1.0, Available platforms: Windows , Linux      |
-| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management privileged user access control and detailed data access audit logging. This protects data wherever it resides, on- premises, across multiple clouds and within big data, and container environments.   |
\ No newline at end of file
+| **Version & Platform**   |  7.1.0, Available platforms: Windows, Linux      |
+| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management, privileged user access control and detailed data access audit logging. This protects data wherever it resides, on-premises, across multiple clouds and within big data, and container environments.   |
```

Comment:
```suggestion
| **Product Description**      |    CipherTrust Transparent Encryption delivers data-at-rest encryption with centralized key management, privileged user access control, and detailed data access audit logging. This protects data wherever it resides: on-premises, across multiple clouds and within big data, and container environments.   |
```

Reply:
I would introduce the short name at first instance of the full product name. Then you can use the short name going forward.